jujugui/templates/index.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<!-- !IE will be true for all non-IE browsers and IE10 since it does not
+ recognize conditional flags. -->
+<!--[if !IE]><!--><script>
+if (/*@cc_on!@*/false) {
+  // Only IE10 has cc_on as false.
+  document.documentElement.className+=' ie10';
+}
+</script><!--<![endif]-->
+<head>
+    <title>Juju GUI</title>
+    <!-- Disable backwards compatible mode for IE on an intranet.
+         For an explanation see http://bit.ly/14VytlD
+         Also note this must be the first <meta> to appear. -->
+    <meta http-equiv="x-ua-compatible" content="IE=edge">
+    <meta charset="utf-8">
+<!-- The license statement comes after the charset declaration so that the
+     charset is delivered within the first 1024 characters. -->
+<!--
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2012-2013 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+    <meta name="viewport" content="width=device-width,initial-scale=1.0 maximum-scale=1.0, user-scalable=no">
+    <meta name="description" content="">
+    <meta name="author" content="Juju team">
+    <script type="text/javascript">
+      // This is used as part of Juju GUI automated QA.
+      window.jsErrors = [];
+      window.onerror = function(msg, source, line) {
+        window.jsErrors.push(msg + ' (' + source + ' at line ' + line + ')');
+      }
+    </script>
+
+    <!-- Make sure the config is loaded before other JS for use in the page
+         below.
+    -->
+    <script type="text/javascript">
+        window.GUI_VERSION = {'version': '', 'commit': ''};
+    </script>
+    <link rel="shortcut icon" href="../static/gui/build/app/favicon.ico">
+
+    <!--[if lt IE 9]>
+    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+
+    <!-- Set up Google analytics async tracking. -->
+    <script type="text/javascript">
+      var _gaq = _gaq || [];
+      window._gaq = _gaq;
+      // Do not call _setAccount as we need to get the key from
+      // the config file which is not yet available.
+    </script>
+  </head>
+  <body>
+    <div style="display: none;" id="svg-sprite"></div>
+    <script src="../static/gui/src/app/assets/js/svg.js"></script>
```

Comment:
Should we instead generate this template with this in it? Especially now that we have 3 templates :)